helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -212,9 +212,28 @@ agent:
   enabled: true
 
   # This Flag enables k8s control plane metric collection.
-  # Details about control plane monitoring and related configurations are located in docs/advanced-configuration.md
+  # Details about control plane monitoring and related configurations are
+  # located in docs/advanced-configuration.md
   controlPlaneEnabled: true
```

Comment:
The name sounds a bit confusing. k8s cannot work without control plane, so it's always enabled. Can we rename it?

What about the following?

```yaml
controlPlaneMetrics:
  coredns:
    enabled: true
  controller-manager:
    enabled: true
  ...
  etcd:
    enabled: false
    clientCert:
    clientKey:
    caFile:
```

etcd certificates must be optional in case user wants to set them up manually. we should rely on `controlPlaneMetrics.etcd.enabled` instead

Reply:
This change has been made.